php/EE/Runner.php

```diff
@@ -741,7 +739,7 @@ public function route() {
 		$site_name = \EE\SiteUtils\auto_site_name( $final_args, $command, $function, $arg_position )[ $arg_position ];
 
 		// Getting the type of the respective site-name from database.
-		$type = EE::db()::get_site_command( $site_name );
+		$type = Site::find( $site_name )->site_type;
```

Comment:
There' a function called above - `auto_site_name`. It ensures site is present in database. If it isn't it will throw error. Hence `Site::find()` will never return `false`.